src/frontend/apps/impress/src/components/DropdownMenu.tsx

```diff
@@ -11,8 +11,7 @@ import { css } from 'styled-components';
 
 import { Box, BoxButton, BoxProps, DropButton, Icon, Text } from '@/components';
 import { useCunninghamTheme } from '@/cunningham';
-
-import { useDropdownKeyboardNav } from './hook/useDropdownKeyboardNav';
+import { useDropdownKeyboardNav } from '@/features/docs/doc-tree/hooks/useDropdownKeyboardNav';
```

Comment:
A generic component should avoid to import a feature component, `useDropdownKeyboardNav` should maybe go to `src/hooks` wdyt ?